src/buffer_logger/can_buffer_logger.py

```diff
@@ -45,11 +49,34 @@ async def listen_to_network(self):
         try:
             while True:
                 can_msg = await self.reader.get_message()
-                # Sleeps 0.1 seconds so doesn't try to read all messages 
+                # Sleeps 0.1 seconds so doesn't try to read all messages
                 # at once.
                 await asyncio.sleep(0.1)
                 self.buffered_data[index] = can_msg
+
                 index = (index + 1) % self.can_logging_buffer
+                # At every n-th index update shared memory for HTTP server
+                if (index % self.can_logging['update_shm'] == 0):
+                    # First close open shared memory
+                    if(self.shm is not None):
+                        self.shm.shm.close()
```

Comment:
wat is shm? shared memory? misschien beter shmem of sharedmem?
Waarom is dit dubbel op? is ook een beetje confusing
```suggestion
                        self.shmem.shmem.close()
```

Reply:
aan de attribuut van self.shm kan ik niks aan doen maar ik zal self.shm het veranderen naar self.shared_list